src/main/java/mod/gate/events/ActionbarReceiveEvent.java

```diff
@@ -0,0 +1,27 @@
+package mod.gate.events;
+
+import net.minecraft.text.Text;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+public class ActionbarReceiveEvent {
+    public Text text;
+    public boolean tinted;
+    public CallbackInfo info;
+
+    public int timeRemaining;
+
+    public String content;
```

Comment:
Just push content up beside time remaining, they are both extra params we added for utility, and maybe think of including a Pattern to match the action bar to so we can get health and mana easily along with skill exp since why not

Reply:
Example

`public class ActionBarMixin {`
`public final Pattern ActionBarParser = new Pattern("idk I don't play skyblock, but make capture groups so you can match and then get the part you want")`

`// the info and other fields here`

`public int timeRemaining;`
`public String content`